src/app/components/MediaLoader/configs/legacyMedia.ts

```diff
@@ -0,0 +1,45 @@
+import filterForBlockType from '#lib/utilities/blockHandlers';
+import {
+  LegacyMediaBlock,
+  ConfigBuilderProps,
+  ConfigBuilderReturnProps,
+} from '../types';
+import buildPlaceholderConfig from '../utils/buildPlaceholderConfig';
+
+export default ({
+  blocks,
+  basePlayerConfig,
+  translations,
+}: ConfigBuilderProps): ConfigBuilderReturnProps => {
+  const legacyMediaBlock: LegacyMediaBlock =
+    filterForBlockType(blocks, 'legacyMedia') ?? {};
+
+  const { image, format, playlist } = legacyMediaBlock?.content ?? {};
+
+  const placeholderConfig = buildPlaceholderConfig({
+    title: '',
+    type: format || 'video',
+    holdingImageURL: image?.href,
+    translations,
+  });
+
+  const mp4Items = playlist
```

Comment:
Are we only supporting mp4s - AFAICR we didn't filter out any items from the list, and let the SMP worry about it (but we did do some other tranformation - see https://github.com/bbc/morph-modules/blob/master/news-av-player/src/legacy-media/transformPlaylist/index.js - not sure if this is still required?)

Reply:
I'll dig out the ticket, but I seen that the older 3gp format is no longer supported, so it would be best to remove these completely from the playlist.

Reply:
Yeah that makes sense. Happy for us to keep this as is.

Reply:
Kinda relates to https://jira.dev.bbc.co.uk/browse/WSTEAM1-249, which is a bit old now but seems to suggest the 3gp format isn't used anymore.